ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -2915,6 +2915,9 @@ literals:
   - LOW
   - MEDIUM
   - HIGH
+- remaining_battery_time_sensor:
```

Comment:
use existing field `remaining_charge_time_sensor` instead

Reply:
Does "remaining_charge_time_sensor" indicate the amount of time remaining until the battery is fully charged, or the amount of time remaining until the battery is fully discharged?  I took it to mean the former, which leaves an important value out of the ontology, being the amount of time the UPS will continue to power the given load in battery mode.